File: totosuki/ABC/A/229.py
S1 = input()
S2 = input()
if (S1[0]+S2[0] == "##") or (S1[1]+S2[1] == "##"):
  print("Yes")
elif S1 == "##" or S2 == "##":
  print("Yes")
else:
  print("No")

File: OCH_Tao/ABC/A/229.js
const [S1, S2] = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
if ((S1 == "#." && S2 == ".#") || (S1 == ".#" && S2 == "#.")) {
  console.log("No");
} else {
  console.log("Yes");
}